docs/en-US/index.md

```diff
@@ -1,202 +1,37 @@
-
-* **Contributing**: Contributions are more than welcome. Please see our current [contributing guidelines](https://github.com/Varying-Vagrant-Vagrants/VVV/blob/master/CONTRIBUTING.md). Thanks!
+[How to Contribute](contributing.md)
 
 ## Table of Content ##
 * [Overview](#overview)
```

Comment:
Removed in https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1112/commits/ed9ae7527591a46b19e7c41de1c7aae8148b0c33